core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerServiceSpec.js

```diff
@@ -0,0 +1,344 @@
+    var expectedNewCustomizationArgs = {
+      choices: {
+        value: [
+          '<p>Option 3</p>'
+        ]
+      }
+    };
+    simpleEditorManagerService.tryToInit();
+    simpleEditorManagerService.saveCustomizationArgs(
+      testStateName, expectedNewCustomizationArgs);
+    expect(explorationStatesService.getState(testStateName)
+      .interaction.customization_args).toEqual(expectedNewCustomizationArgs);
+  });
+  /**
+    * Not working, needs debugging.
+    It('should save the answer groups', function () {
+      var testStateName = 'Introduction';
+      var expectedNewAnswerGroups = [{
+        rule_specs: [{
+          rule_type: 'Equals',
+          inputs: {
+            x: 1
+          }
+        }]
+      }];
+      simpleEditorManagerService.tryToInit();
+      simpleEditorManagerService.saveAnswerGroups(
+        testStateName, expectedNewAnswerGroups);
+      expect(explorationStatesService.getState(testStateName)
+        .interaction.answer_groups).toContain(expectedNewAnswerGroups);
+    });
+  */
+
+  it('should save the default outcome', function() {
+    var testStateName = 'Introduction';
+    var expectedNewDefaultOutcome = {
+      dest: 'Introduction',
+      feedback: [
+        '<p>One is wrong bcz ofisEmpty many reasons</p>'
+      ]
+    };
+    simpleEditorManagerService.tryToInit();
+    simpleEditorManagerService.saveDefaultOutcome(
+      testStateName, expectedNewDefaultOutcome);
+    expect(explorationStatesService.getState(testStateName)
+      .interaction.default_outcome).toEqual(expectedNewDefaultOutcome);
+  });
+
+  /**
+    * Not working, needs debugging.
+  it('should save the bridge html', function () {
+    var testStateName = 'Introduction';
+    var expectedNewBridgeHtml = '<p> lets move to next Qsn </p>';
+    var expectedSimpleEditorManagerQuestionList = questionList;
+    simpleEditorManagerService.tryToInit()
+    simpleEditorManagerService.saveBridgeHtml(
+      testStateName, expectedNewBridgeHtml);
+    expect(explorationStatesService
+      .getState(expectedSimpleEditorManagerQuestionList
+        .getNextStateName(testStateName)).content)
+          .toEqual(expectedNewBridgeHtml);
+  });
+*/
+  it('should add new question and new state', function() {
+    var expectedSimpleEditorQuestionList = questionList;
+    var expectedLastStateName;
+    var expectedDefaultOutcome;
+    var DEFAULT_INTERACTION = {
+      ID: 'MultipleChoiceInput',
+      CUSTOMIZATION_ARGS: {
+        choices: {
+          value: ['<p>Option 1</p>']
+        }
+      }
+    };
+
+    simpleEditorManagerService.tryToInit();
+    expectedLastStateName =
+      expectedSimpleEditorQuestionList.getLastQuestion()
+        .getDestinationStateName();
+
+    expectedDefaultOutcome = {
+      dest: expectedLastStateName,
+      feedback: [''],
+      param_changes: []
+    };
+
+    simpleEditorManagerService.addNewQuestion();
+    expect(explorationStatesService.getState(expectedLastStateName)
```

Comment:
You could define an intermediate variable:

    var state = explorationStatesService.getState(expectedLastStateName);

Saves repetition.